index.bs

```diff
@@ -101,7 +101,7 @@ name.
 <h3 id="monkeypatch-navigation">Navigation</h3>
 
 This section ensures that an [=attribution source=] associated with a navigation
-results in a top-level navigation whose final URL is [=same site=] with the
+results in a top-level navigation whose final URL equals the
```

Comment:
I don't think this is exactly right. The URL itself might not equal the destination.